doc/src/conf.py

```diff
@@ -495,6 +495,18 @@
     "https://github.com/{}/sympy/blob/{}/sympy/".format(fork, commit_hash)
 
 
+# Configuration for interactive documentation using jupyterlite-sphinx and Pyodide
+# See https://jupyterlite-sphinx.readthedocs.io/en/stable/ for options
+
+global_enable_try_examples = True
+try_examples_global_button_text = "Try it!"
+try_examples_global_warning_text = (
+    "Interactive examples for SymPy are experimental and may not always "
+    "work as expected. If you encounter any issues, please report them on "
+    "the [SymPy issue tracker](https://github.com/sympy/sympy/issues/new)."
+)
```

Comment:
I'd like to ask if we should open a tracking issue whose purpose would be solely to garner feedback on interactive docs in general, and replace the link with it here? I imagine it would be better if the feedback we receive is consolidated in one place, i.e., in one single issue rather than in multiple issues shall users open any.

Reply:
Yes, let's do that. We can split things off into separate issues from there.

Reply:
I opened `https://github.com/sympy/sympy/issues/27657` (enclosed this issue backticks so to not link both ways) and updated the link via 92f24f3feefacc2117e901ee1f42519f65861af5. Any suggestions on the copy there are very much welcomed (please feel free to edit the issue body directly, if you need to).